tests/unit/test_charm.py

```diff
@@ -31,7 +32,7 @@ def raise_url_error(*args, **kargs):
     raise urllib.error.URLError("mock error")
 
 
-def mock_get_latest_runner_bin_url():
+def mock_get_latest_runner_bin_url(os_name: str = "linux", arch: ARCH = ARCH.X64):
```

Comment:
Would it be possible and valuable to also have an arm test here?

Reply:
Added a test for downloading runner bin!